core/src/forwarding_stage.rs

```diff
@@ -550,7 +587,7 @@ impl ForwardingStageMetrics {
                     i64
                 ),
                 ("votes_forwarded", metrics.votes_forwarded, i64),
-                ("non_votes_received", metrics.non_votes_received, i64),
+                ("votes_forwarded", metrics.votes_forwarded, i64),
```

Comment:
do we have votes_forwarded metric twice?

Reply:
yeah, it looks preceding problem and than I copy-pasted for non-votes. Fixed.